Documentation/Rust Style.md

```diff
@@ -36,6 +36,31 @@ Apply `rustfmt` to new code before committing, using the default configuration o
 
 In `impl`s, always put public functions before private ones.
 
+## Bringing names into scope (`use` statements)
+
+Generally `use` statements should be employed to bring names from different modules into scope. However, functions from other modules should not be brought fully into scope. Instead their module should be brought into scope meaning that subsequent usage of the function requires one level of qualification. For example, if we have:
+
+```rust
+pub mod a {
+    pub mod b {
+        pub struct Harbour {}
+        pub fn bar() {}
```

Comment:
If it were `Harbour::bar()`, however, then it should have gone into the std and default imported like `Vec<_>`